akka-http-core/src/main/scala/akka/http/impl/engine/client/PoolSlot.scala

```diff
@@ -38,67 +36,41 @@ private object PoolSlot {
     final case class ConnectedEagerly(slotIx: Int) extends SlotEvent
   }
 
-  /*
-    Stream Setup
-    ============
-
-    Request-   +-----------+              +-------------+              +------------+
-    Context    | Slot-     |  List[       |   flatten   |  Processor-  | SlotEvent- |  Response-
-    +--------->| Processor +------------->| (MapConcat) +------------->| Split      +------------->
-               |           |  Processor-  |             |  Out         |            |  Context
-               +-----------+  Out]        +-------------+              +-----+------+
-                                                                             | RawSlotEvent
-                                                                             | (to Conductor
-                                                                             |  via slotEventMerge)
-                                                                             v
-   */
-  def apply(slotIx: Int, connectionFlow: Flow[HttpRequest, HttpResponse, Any])(implicit system: ActorSystem, fm: Materializer): Graph[FanOutShape2[SlotCommand, ResponseContext, RawSlotEvent], Any] =
-    GraphDSL.create() { implicit b ⇒
-      import GraphDSL.Implicits._
-
-      val slotProcessor = b.add(Flow.fromGraph(new SlotProcessor(slotIx, connectionFlow, system.log)).mapConcat(ConstantFun.scalaIdentityFunction))
-      val split = b.add(Broadcast[ProcessorOut](2))
-
-      slotProcessor ~> split.in
-
-      new FanOutShape2(
-        slotProcessor.in,
-        split.out(0).collect { case ResponseDelivery(r) ⇒ r }.outlet,
-        split.out(1).collect { case r: RawSlotEvent ⇒ r }.outlet)
-    }
+  def apply(slotIx: Int, connectionFlow: Flow[HttpRequest, HttpResponse, Any])(implicit system: ActorSystem /*REMOVEME*/ , m: Materializer): Graph[FanOutShape2[SlotCommand, ResponseContext, RawSlotEvent], Any] = {
```

Comment:
i've been cheating with my testing and needed the binary compatibility so was leaving it in.